src/main/java/duke/messages/Output.java

```diff
@@ -0,0 +1,191 @@
+package duke.messages;
+
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * Contains the relevant responses and messages.
+ */
+public class Output {
+
+    private static final String logo = "     ___   ____    __    ____  _______     _______."
```

Comment:
Would it be better to name constant (static final) fields with upper case letters?